g5k_discovery/vue/HardwareCatalogue.vue

```diff
@@ -183,14 +248,61 @@ export default {
       filteredNodes: [],
       visibleNodes: [],
       reservationProperties: [],
+      calendarLinks: {
```

Comment:
We should be using the site base URLs and names from the hardware API `/sites` response if possible. Having to configure it here will make it difficult to add/remove a site if needed.